### ISSUE TITLE
Rename and register

### ISSUE THREAD

**Issue by MaximeBouton** [tawheeler/Vec.jl#14]
It would be nice to register this package in the general registry, it would also be required if we want to register https://github.com/sisl/AutomotiveDrivingModels.jl 

I think a necessary condition to register this package is to change the name, it is too generic.

Name Suggestions: 
- Coordinates.jl 
- ...

**Comment by rejuvyesh** [comment]
`Coordinates.jl` seems appropriate to me.

**Comment by rejuvyesh** [comment]
Although, looks like there's a related package: https://github.com/FugroRoames/CoordinateTransformations.jl

**Comment by MaximeBouton** [comment]
@mattuntergassmair suggested `EuclideanGroupVectors.jl`
It is more specific than `Coordinates.jl`

**Comment by mattuntergassmair** [comment]
Yes I feel like `Coordinates.jl` could be anything, I don't think `Vec.jl` covers enough width to have that name. It seems to me that most of the stuff in there is related to Euclidean Groups and Vectors that are part of those groups.
I'm not necessarily attached to `EuclideanGroupVectors.jl` but it seems to me that with the help of a hardcore mathematician / algebraist we may be able to find the common denominator of whatever is implemented. Some sort of "rotational symmetry group" or something.

**Comment by mattuntergassmair** [comment]
mainly quaternions seem to break the pattern since they may not be forming an Euclidean group - the details of that are beyond me

**Comment by MaximeBouton** [comment]
There is also all these: https://github.com/sisl/Vec.jl/blob/master/src/coordinate_transforms.jl
They seem to be useful to Aerospace applications. It is a bunch of coordianate systems. 

The JuliaGeo organization seem to have a much better support for those: 
https://github.com/JuliaGeo/Geodesy.jl

It makes me wonder if this whole package has a reason to live outside of AutomotiveDrivingModels.jl ?

**Comment by mattuntergassmair** [comment]
Sounds like it's a lot of different things together...

**Comment by MaximeBouton** [comment]
yeah :/ For the Automotive stuff, the useful components are `VecE2`, `VecSE2` and all the geometry functions.
